nni/experiment/config/convert.py

```diff
@@ -14,7 +14,7 @@
 
 
 def to_v1_yaml(config: ExperimentConfig, skip_nnictl: bool = False) -> Dict[str, Any]:
-    config.validate(skip_nnictl)
+    config.validate(False)
```

Comment:
`skip_nnictl` is not used in this function. what is the meaning of `skip_nnictl`?

Reply:
This function is likely to be removed soon. I'm too lazy to make it elegant.